install/kubernetes/helm/subcharts/pilot/values.yaml

```diff
@@ -22,3 +22,14 @@ env:
 cpu:
   targetAverageUtilization: 80
 nodeSelector: {}
+# Provide options to allow user to configure JWT related fields.
```

Comment:
I'm not sure have this under pilot is a right choice. + @sdake, do we have convention where the helm values should be?